css/home.css

```diff
@@ -0,0 +1,436 @@
+/* Generated by http://css.spritegen.com CSS Sprite Generator */
+ 
+.sprite-bottle-small, .sprite-bottle-smallest, .sprite-facebook, .sprite-twitter, .sprite-menu, 
+.sprite-google, .sprite-more, .sprite-ul
+{ display: inline-block; background: url('../images/sprites.png') no-repeat; overflow: hidden; text-indent: -9999px; text-align: left; }
+ 
+.sprite-bottle-small { background-position: -0px -0px; width: 250px; height: 250px; }
+.sprite-bottle-smallest { background-position: -0px -250px; width: 160px; height: 160px; }
+.sprite-facebook { background-position: -160px -250px; width: 25px; height: 25px; }
+.sprite-twitter { background-position: -185px -250px; width: 25px; height: 25px; }
+.sprite-menu { background-position: -210px -250px; width: 24px; height: 24px; }
+.sprite-google { background-position: -210px -274px; width: 22px; height: 22px; }
+.sprite-more { background-position: -234px -250px; width: 12px; height: 13px; }
+.sprite-ul { background-position: -234px -263px; width: 12px; height: 13px; }
+
+/* common */
+* {
+	text-decoration: none;
+}
+
+body {
+	margin: 5px;
+	background-color: #EAE2D7;
+}
+
+p {
+	color: #8F816C;
+	font-size: 0.8em;
+}
+
+.bold {
+	font-weight: bold;
+}
+
+/* clearfix for modern browsers */
+.clearfix:before,
+.clearfix:after {
+   content: "";
+   display: table;
+}
+.clearfix:after {
+   clear: both;
+}
+ 
+/* clearfix for IE6-7 */
+.clearfix {
+   zoom: 1;
+}
+
+
+.sprite-bottle-small {
+	display: none;
+}
+
+.sprite-bottle-smallest {
+	float: right;
+	display: inline;
+}
+
+/* logo */
+#logo {
+	text-align: center;
+}
+
+#logo h1 {
+	color: #E61052;	
+	font-size: 2em;
+	margin: 0;
+}
+
+#logo p {
+	color: #2A1F0F;
+	font-size: 0.5em;
+	margin: 0;
+}
+
+
+/* logo-banner */
+#logo-banner {
+	display: none;
+	
+}
+
+#logo-banner h2, 
+#logo-banner p {
+	transform: rotate(-7deg);
+	color: #FFF;
+	margin: 0px;
+}
+
+#logo-banner h2 {
+	margin-left: 0.3em;
+	font-size: 1.5em;
+	opacity: 0.6;
+}
+
+#logo-banner p {
+	font-size: 1em;
+	opacity: 0.8;
+}
+
+
+/* search */
+#search {
+	font-size: 0.8em;
+	text-align: center;
+	margin: 0.5em;
+}
+
+#search input ,
+#search button {
+	border-radius: 8px;
+	border-width: 0;
+	padding: 8px;
+	behavior: url(./plugins/PIE/PIE.htc);
+}
+
+#search button[type="submit"] {
+	background-color: #E61052;
+	color: #FFF;
+}
+
+
+/* main-menu */
+.sprite-menu {
+	float: left;
+}
+
+#main-menu {
+	position: absolute;
```

Comment:
You don't seem to have a parent container.
Maybe 100% width would be better for mobile?